src/react/components/_dashboard/UpNext/UpNextItem.js

```diff
@@ -25,14 +25,15 @@ const UpNextItem = ({
 }) => {
   const ga = new googleAnalytics(applicationLayer);
 
-  const expandItem = (index, setExpandedItemIndex) => {
-    setExpandedItemIndex(index);
+  const toggleItem = (index, expandedItemIndex, setExpandedItemIndex) => {
```

Comment:
You can greatly simplify toggleItem because all the items are props passed into the UpNextItem

const toggleItem = () => {
  setExpandedItemIndex((index == expandedItemIndex) ? null : index)
  ga.sendEvent('Detailed view', showItem ? 'Open' : 'Close', 'Up Next');
}

And then obviously the callers don't need the arguments